### PR TITLE
I18n login

### PR DESCRIPTION
**Pull request by rutvik-balar**
**Task**
I18 in auth
 
**Todo**
 
add i18n into login
add i18n into register 
 
**How to check**
npm run dev

**Evidences**
 
Screenshot 2024-01-02 at 10 13 12 AM
Screenshot 2024-01-02 at 10 13 54 AM

 
**build**
**Related Links**

### CODE REVIEW

File: src/config/translations/locales/en.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// en.js
+
+export const en = {
+  page: {
```
Comment (r-aritra) [inline src/config/translations/locales/en.ts:4]: make the features as keys

File: src/features/Login/Login.tsx [hunk 2]
```diff
@@ -59,25 +62,25 @@ export default function Login() {
...
               required
-              label="Email"
-              placeholder="hello@mantine.dev"
+              data-testid="email-input"
+              label={t('page.login.emailLabel')}
+              placeholder={t('page.login.emailPlaceholder')}
               value={form.values.email}
               onChange={(event) => form.setFieldValue('email', event.currentTarget.value)}
-              error={form.errors.email && 'Invalid email'}
+              error={form.errors.email && t('page.login.page.login.invalidEmailError')}
```
Comment (r-aritra) [inline src/features/Login/Login.tsx:70]: wrong i18n string

**Review by r-aritra — CHANGES_REQUESTED** [review]

**Comment by r-aritra** [comment]
Try to add eslint rules for correct i18n string

File: src/config/translations/locales/en.ts (hunk 1 above)
Comment (rutvik-balar) [reply src/config/translations/locales/en.ts]: resolved

File: src/features/Login/Login.tsx (hunk 2 above)
Comment (rutvik-balar) [reply src/features/Login/Login.tsx]: resolved

**Review by r-aritra — APPROVED** [review]
lgtm

**Pull request merged**